pcweb/components/docpage/sidebar/sidebar.py

```diff
@@ -266,42 +266,49 @@ def get_prev_next(url):
     return None, None
 
 
-def sidebar_category(name, icon, color, index):
-    return rx.flex(
-        rx.button(
-            rx.icon(
-                tag=icon,
-                color="white",
-                fill="rgba(255, 255, 255, 0.25)",
-                size=20,
+def sidebar_category(name, url, icon, color, index):
+    return rx.link(
+        rx.flex(
+            rx.button(
+                rx.icon(
+                    tag=icon,
+                    color="white",
+                    fill="rgba(255, 255, 255, 0.25)",
+                    size=20,
+                ),
+                height="35px",
+                width="35px",
+                padding="0px",
```

Comment:
Is this needed? Does button set default padding?

Reply:
This was same as before just refactored the function yes its needed we use more than the default spacing